gulpfile.js

```diff
@@ -50,7 +50,7 @@ var filePath = {
     },
     styles: {
         src: './app/app.less',
-        watch: ['./app/app.less', './app/**/*.less', './app/components/**/*.less']
+        watch: ['./app/app.less', './app/**/*.less', './app/components/common/**/*.less', './app/components/**/*.less']
```

Comment:
This shouldn't be necessary, the `./app/**/*.less' rule should take care of components and components/common.